### ISSUE TITLE
VR Headtracking fails when table uses an undefined LUT as default

### ISSUE THREAD

**Issue by gjupp** [vpinball/vpinball#1741]
**Describe the bug**
Several tables that feature multiple LUT options break the headtracking on VR (experienced on Quest 2, Wireless via Virtual Desktop). Headtracking is paused and entire VR image stays at a fixed viewpoint regardless of what direction the head is orientated.

Examples of tables include:
Class of 1812 - https://vpuniverse.com/files/file/8152-class-of-1812-gottlieb-1991-w-vr-room/
Cyclone - https://vpuniverse.com/files/file/18687-cyclone-williams-1988-v211-dt-fs-fss-vr/
There was also a report of Die Hard also being affected but only when ball passes over right ramp, but I haven't personally tested that table. There may be other tables also facing the same issue.

It had been noted that there were several ways to overcome the problem.
1) Set playfield reflection strength to 0
2) Or switch to to a different LUT (the Color Saturation percentage feature in some instances).

I had a look through the script and noticed that the default LUT was undefined on first startup for both the tables linked above. For instance for Cyclone table the script is as follows:

```
	'*********************
	'Color LUT
	'*********************

	if ColorLUT = 1 Then Table1.ColorGradeImage = ""
	if ColorLUT = 2 Then Table1.ColorGradeImage = "LUT0"
	if ColorLUT = 3 Then Table1.ColorGradeImage = "LUT1"
	if ColorLUT = 4 Then Table1.ColorGradeImage = "LUT2"
	if ColorLUT = 5 Then Table1.ColorGradeImage = "LUT3"
	if ColorLUT = 6 Then Table1.ColorGradeImage = "LUT4"
	if ColorLUT = 7 Then Table1.ColorGradeImage = "LUT5"
	if ColorLUT = 8 Then Table1.ColorGradeImage = "LUT6"
	if ColorLUT = 9 Then Table1.ColorGradeImage = "LUT7"
	if ColorLUT = 10 Then Table1.ColorGradeImage = "LUT8"
	if ColorLUT = 11 Then Table1.ColorGradeImage = "LUT9"
	if ColorLUT = 12 Then Table1.ColorGradeImage = "LUT10"
	if ColorLUT = 13 Then Table1.ColorGradeImage = "LUT11"
	if ColorLUT = 14 Then Table1.ColorGradeImage = "LUT12"
	if ColorLUT = 15 Then Table1.ColorGradeImage = "LUT13"
	if ColorLUT = 16 Then Table1.ColorGradeImage = "LUT14"
	if ColorLUT = 17 Then Table1.ColorGradeImage = "LUT15"
```

If I instead change the script to refer to a particular LUT then headtracking starts working. ie:
`if ColorLUT = 1 Then Table1.ColorGradeImage = "LUT0"`

Likewise, Class of 1812 also has a missing definition for the default LUT in the script and defining a LUT for the default also fixed the issue on that table.

Interestingly the issue can also be temporarily fixed by switching to any other available LUT, then even if you switch back to the default LUT the headtracking still works for that session. But this only works during the current session and must be reapplied each time game is played unless the setting is saved.

**Expected behavior**
This appears fixable by a script edit, but ideally it would be good if headtracking were not broken if a default LUT is undefined.

**Versions & Settings**
 - VPX: [10.8.0 Build 1953 ddcc89d GL 64
 - Windows version 10
 - GTX 1080ti

**Comment by gjupp** [comment]
I've noticed that Defender VPW v1.3 also has VR headtracking issues for me when using the segment display option, headtracking runs extremely slowly but doesn't seem to completely fail like Class of 1812 and Cyclone. Oddly, it seems fine if the backglass option is switched to DMD mode though.

Today I installed Tornado Rally (original table by Eric3dee) which also has headtracking issues. It seems to be related to something in the attract mode as if I start a game quickly as soon as it loads it seems ok, but if I wait a little while then it loses headtracking and freezes for a little while, (or after a game finishes) sometimes it's a permanent freeze.

**Comment by gjupp** [comment]
I've recently had a chance to play Die Hard Trilogy and can confirm that the VR headtracking issue also exists in this table due to lacking a default LUT.

Issue still exists in 10.8 2051 Final.

Is it looking to be addressed in 10.8.1 ? I haven't tried that version yet.

**Comment by gjupp** [comment]
Time Machine v2.1 by RobbyKingpin added to the list of tables with this issue. https://vpuniverse.com/files/file/24756-time-machine-data-east-1988-v21

**Comment by Joey-2001** [comment]
I just download Time Machine 2.1 and it works without issue in VR.
10.8.0-2051-VPM 3.7.0.3 x64

**Comment by gjupp** [comment]
> I just download Time Machine 2.1 and it works without issue in VR. 10.8.0-2051-VPM 3.7.0.3 x64

Yes, as I stated in the opening post the issue does not occur for all users and is perhaps related to lower end PCs becoming swamped whilst higher powered PCs brute force their way through the issue.

What are your PC specs?

**Comment by gjupp** [comment]
Defender VPW v1.3 also has VR headtracking issues when using the segment display option (UseFlexDMD = False). It seems fine in FlexDMD mode though. This can be fixed by defining a LUT in the Backglass View mode of the editor (none is defined initially).

I suspect that this table may be a good candidate to track down the true cause of the issue given that it only rears it's head when the segment display mode is selected in the script.
